installer/scripts/omsadmin.sh

```diff
@@ -632,7 +636,7 @@ onboard()
         cleanup_certs
         return $INVALID_PROXY
     elif [ $error -ne 0 ]; then
-        log_error "Error during the onboarding request. Check the correctness of the workspace ID and shared key or run omsadmin.sh with '-v'"
+        log_error "Error during the onboarding request: curl returned $error. Check the correctness of the workspace ID and shared key or run omsadmin.sh with '-v'"
         cleanup_certs
         return $ERROR_ONBOARDING
```

Comment:
I thought I saw somewhere that there are curl specific errors.  Are those pertinent here?  Perhaps even to embed in the log message if not to return.

Reply:
You can see the curl-specific error codes here: https://help.github.com/articles/testing-your-ssh-connection/

If you look in the lines of code above these, we check if any of those curl-specific error codes apply; if not, we fall to this case. This changed line is just adding curl's return code to the printout for diagnostic purposes.